Git actions are failing for python 3.10

<img width="879" alt="image" src="https://user-images.githubusercontent.com/37095187/107108733-101b4780-67ef-11eb-9468-37479136b7c3.png">

I see 😞 . Python 3.10 should be allowed to fail, since it is not yet release; we should use it as just a heads up for potential issues for supporting py3.10. Can you post a link to the documentation, bug report, etc, where u found what is being removed? From the stacktrace it looks like the way enums are implemented is being changed.

Where in the stacktrace the error is... "it depends". In async it can pop up in different place. Also exceptions can be thrown when handling exceptions. In the simpler cases, as this one, it is typically found at the bottom of the stacktrace output.
```
 /opt/hostedtoolcache/Python/3.10.0-alpha.5/x64/lib/python3.10/enum.py:146: in __get__
    raise AttributeError(
E   AttributeError: Extra: no attribute 'allow'
```
If you follow the stacktrack you can see that it goes up to:
```
 fizzbuzzx_api.py:6: in <module>
    from fastapi import FastAPI
```
So since this is where we are importing FastAPI, the issue is outside our code and with a dependency. We can investigate workarounds, check the FastAPI roadmap/issues. If this has not been reported to FastAPI, this would be a great opportunity to contribute to an upstream project.

Do you understand what the stack is? I recommend doing a quick refresher on it.

--- a/.github/workflows/tox_tests.yml
+++ b/.github/workflows/tox_tests.yml
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [3.6,  3.7, 3.8, 3.9, 3.10-dev]
+        python: [3.6,  3.7, 3.8, 3.9]
 
     steps:
       - uses: actions/checkout@v2

